Exception/Uploadable/InvalidFilesystem.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * (c) FSi sp. z o.o. <info@fsi.pl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FSi\Bundle\DoctrineExtensionsBundle\Exception\Uploadable;
+
+use FSi\DoctrineExtensions\Uploadable\Exception\RuntimeException;
+
+class InvalidFilesystem extends RuntimeException
```

Comment:
Exception class name should always ends with 'Exception', so `InvalidFilesystemException`